### DISCUSSION TITLE
Is it passible to set an encrypted password to database when new database creates in windows

### DISCUSSION THREAD

**Discussion by iampopal** [simolus3/drift#1248] [General]
I am using the moor package for a windows desktop application, I need to secure the database file so other people shall not able to get data from the database file. 
Is it possible to set a password to database file when a new database creates.

**Comment by simolus3** [comment]
It looks like you've already seen [the encrypted version of moor/ffi](https://moor.simonbinder.eu/docs/other-engines/encryption/#encrypted-version-of-moorffi) which you can use here. I'll take a look at your issue with the opening override too.
